src/NorthwindWebsite/Configuration/RoutingConfiguration.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace NorthwindWebsite.Configuration;
+
+public static class RoutingConfiguration
+{
+    public static void ConfigureRouting(this WebApplication app)
+    {
+        app.UseEndpoints(endponts =>
+        {
+            endponts.MapControllerRoute(
+                name: "default",
+                pattern: "{controller=Home}/{action=Index}/{id?}");
+
+            endponts.MapControllerRoute(
```

Comment:
Since this is a specific configuration for specific controller let's put it before base one